layouts/_default/baseof.html

```diff
@@ -14,6 +14,11 @@
       <meta name="robots" content="index, follow">
     {{ end }}
 
+    {{ block "meta-tags"}}
+      <meta name="description" content="{{ $.Summary | default "Zit je ergens vast bij wiskunde of fysica? Met Hoe Zit Het? heb je de leerstof razendsnel onder de knie. Neem maar eens een kijkje naar onze lessen. Ze zijn kort, krachtig, vol illustraties en bovenal helemaal gratis!"}}"/> 
```

Comment:
Wanneer een template in Hugo wordt gerenderd, krijgt die template een "context" mee waar je aan kan via `.`. Hier was `{{ .Summary | default "......" }}` dus voldoende geweest.
Het dollarteken hoef je enkel te gebruiken wanneer `.` niet meer de context bevat die je nodig hebt. Dat kan bijvoorbeeld gebeuren wanneer je een [itereert over een array](https://gohugo.io/templates/introduction/#iteration) met een soort `for each`-lus, want binnen zo'n lus krijgt `.` de huidige waarde van het element in de array. Bijvoorbeeld:
```html
{{ range .Params.tags }}
    {{ . }}  <!-- Hier bevat `.` de waarde van een tag -->
    {{ $.Summary }}   <!-- De enige manier om nu aan de summary te geraken is via het dollarteken -->
{{ end }}
```

Reply:
Aha, dat verduidelijkt het! Had daar niet echt achter gezocht. Dit werkt dus wel, maar 't is beter (of in ieder geval korter) zonder de '$'?

Reply:
Beter omdat het korter is inderdaad 😃